app/economy/management/commands/get_prices.py

```diff
@@ -177,6 +178,57 @@ def cryptocompare():
         refresh_conv_rate(obj.created_on, obj.subscription.token_symbol)
 
 
+def uniswap():
+    """Hangle pulling market data from Uniswap using its subgraph node on mainnet."""
+    endpoint = 'https://api.thegraph.com/subgraphs/name/graphprotocol/uniswap'
+    # Get the top-50 exchange pairs in terms of total ETH locked
+    # See https://github.com/graphprotocol/uniswap-subgraph#important-info
+    query_limit = 50
```

Comment:
Since the `decimals` calls work for all ERC20s in Uniswap now, can we remove the query limit to pull in all Uniswap prices? Or change the limit to pull in the top 100 exchanges instead?